<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>536</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Runtime UI Serialize Test</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QListView" name="listView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="4">
     <widget class="QScrollArea" name="frameEmbedded">
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>566</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLibrary">
    <property name="title">
     <string>Library</string>
    </property>
    <addaction name="actionReadUI"/>
    <addaction name="actionWriteUI"/>
    <addaction name="separator"/>
    <addaction name="actionReadFile"/>
   </widget>
   <addaction name="menuLibrary"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionReadUI"/>
   <addaction name="actionWriteUI"/>
   <addaction name="separator"/>
   <addaction name="actionReadFile"/>
  </widget>
  <action name="actionReadUI">
   <property name="icon">
    <iconset>
     <normaloff>:logo/ico/scanframe</normaloff>:/logo/ico/scanframe</iconset>
   </property>
   <property name="text">
    <string>&amp;Read UI</string>
   </property>
   <property name="toolTip">
    <string>Read UI from file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionWriteUI">
   <property name="icon">
    <iconset>
     <normaloff>:logo/ico/scanframe</normaloff>:logo/ico/scanframe</iconset>
   </property>
   <property name="text">
    <string>&amp;read UI</string>
   </property>
   <property name="toolTip">
    <string>read UI to file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionReadFile">
   <property name="icon">
    <iconset>
     <normaloff>:logo/ico/scanframe</normaloff>:logo/ico/scanframe</iconset>
   </property>
   <property name="text">
    <string>Read &amp;File</string>
   </property>
   <property name="toolTip">
    <string>Read UI from file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
